packages/azure-services/src/key-vault/secret-names.ts

```diff
@@ -5,4 +5,6 @@ export const secretNames = {
     cosmosDbKey: 'cosmosDbKey',
     storageAccountName: 'storageAccountName',
     storageAccountKey: 'storageAccountKey',
+    serverAppRegistrationSecret: 'A11yServiceServerAppRegistration',
```

Comment:
remind me what these are used for?

Reply:
They are used in authentication. In web worker we only need serverAppRegistrationSecret, in the future like e2e test we will use clientAppRegistrationSecret (consumer app).